site/src/docs/configuration/authorization/index.md

```diff
@@ -6,6 +6,23 @@ title: Authorization
 
 Authentication handled by external providers. You should set up OAuth2 for at least one of them to allow users to make comments. It is not mandatory to have all of them, but one should be correctly configured.
 
+### Apple
```

Comment:
maybe we should add "not fully implemented yet", as this thing is not yet working on the frontend side?